example/btrfs-subvolumes.nix

```diff
@@ -45,7 +45,32 @@
                   };
                   # This subvolume will be created but not mounted
                   "/test" = { };
+                  # Subvolume for the swapfile
+                  "/swap" = {
+                    mountpoint = "/.swapvol";
+                    swap.enable = true;
+                    swap.files = [
+                      {
+                        size = "20M";
+                      }
+                      {
+                        size = "20M";
+                        path = "rel-path";
+                      }
+                    ];
+                  };
                 };
+
+                mountpoint = "/partition-root";
+                swap.enable = true;
+                swap.files = [
+                  {
+                    size = "20M";
+                  }
+                  {
+                    size = "20M";
+                  }
+                ];
```

Comment:
```suggestion
                swap.files = {
                  swapfile1 = {
                    size = "20M";
                  };
                  swapfile2 = {
                    size = "20M";
                  };
                };
```

Reply:
how about this schema? lists are pretty bad since you cannot easy modify of reference them. the key could be the default value of the path?

Reply:
Agreed. Also - maybe I should just move the `enable` switch to the individual swap files as well, such that separate modules can disable a specific swap file for instance? I'll try to update it today.

Reply:
with attrsOf we don't need an enable flag. either it's there or it isn't

Reply:
Right - but can you disable it in another module than where it was created? I personally have my disko config split up into multiple modules using things like `mkDefault`, and I don't know how to "remove" a already declared key in an attribute set.

Reply:
Ah there are multiple ways. The easiest would be just to define it where needed. But you use swap.files = lib.mkForce {} if you want to empty it

Reply:
Fair enough. I can remove the enable flag if you think that's better.